src/services/refact.ts

```diff
@@ -154,6 +155,37 @@ export async function getCaps(lspUrl?: string): Promise<CapsResponse> {
   return json;
 }
 
+export function isStatisticDataResponse(
+  json: unknown,
+): json is { data: string } {
+  if (!json || typeof json !== "object") return false;
+  return typeof (json as { data?: unknown }).data === "string";
```

Comment:
can remove the type casting with this
```suggestion
  if ( !("data" in json) ) return false
  return typeof data.json === "string";
```